Dockerfile

```diff
@@ -1,5 +1,5 @@
-FROM node:lts-alpine

```

Comment:
Changing our image away alpine is a bit of a large change if we're just trying to get your env up and running. Did you see somewhere that node-alpine isn't compatible with your machine?